version.go

```diff
@@ -35,6 +35,16 @@ func (version *ArtifactVersion) refresh() {
 	}
 }
 
+// IsStableVersion returns true if the version indicates an official release.
+func (version *ArtifactVersion) IsDevelopment() (bool, error) {
+	parsedVersion, err := CreateMongoDBVersion(version.Version)
+	if err != nil {
+		return false, errors.Wrap(err, "could not parse version")
+	}
+	// handle legacy and modern versioning schemes
+	return !parsedVersion.IsStableSeries() && !parsedVersion.IsLTS(), nil
```

Comment:
Is this supposed to include continuous releases?

Reply:
Good question, I'm assuming yes (and I could just say IsContinuous() instead of !IsLTS() actually) Also because the comment refers to "the 'latest' (e.g. nightly) build archive". In Julian's code we included continuous with nightly builds, and LTS with stable releases.

Reply:
Really though this is probably determined by how we implement changes to mr? Since we use these urls to download the file. We could postpone merging this until we know what's up with that (unless we already know)